uint64 id
string name
string ref_frame
bool active
geometry_msgs/Vector3 translation
geometry_msgs/Quaternion rotation
---
bool success